x-pack/plugins/observability_solution/observability/public/navigation_tree.ts

```diff
@@ -415,12 +415,6 @@ function createNavTree({ streamsAvailable }: { streamsAvailable?: boolean }) {
                   defaultMessage: 'Logs categories',
                 }),
               },
-              {
```

Comment:
**nit:** I think we should also remove the log stream entry above [here](https://github.com/elastic/kibana/pull/203996/files#diff-8f26b8327cc9fc31bef2b22bb53b82256edc9cf05cfc9c766d746a7aa4532437R400-R405) in the same file right?

Reply:
Good spot, removed [here](https://github.com/elastic/kibana/pull/203996/commits/a3ebe872299f1ffb592a38ef78cf05c063e184de)